2016-09/september-21.md

```diff
@@ -0,0 +1,49 @@
+## September 21 ([discuss](https://github.com/webpack/meeting-notes/pull/14))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+
+### Agenda
+
+#### Documentation Updates
+
+* Juho is going to start working on [antwar/antwar#103](https://github.com/antwarjs/antwar/issues/103) next week.
```

Comment:
I can also give a hand, Phenomic [just got support for webpack 2](https://github.com/MoOx/phenomic/releases/tag/0.17.0) (so we support both 1) and is now using [hard-source-webpack-plugin](https://github.com/mzgoddard/hard-source-webpack-plugin) - which is awesome btw!